### PR TITLE
#166880030 Setup and deploy base application to Heroku

### PR DESCRIPTION
**Pull request by Rhytah**
#### What does this PR do?

- Setup and deploy base application to Heroku

- This will enable other users to access and use the application.

#### Description of Task to be completed?
- Update `Procfile` to display our build logs
- Remove older `app.json` file
- Create and set up application pipeline on `Heroku`
- Add Heroku generated app.json

#### How should this be manually tested?
- Go to [staging app](https://ah-backend-kronos-staging.herokuapp.com/)
- You can access a [review-app](https://ah-backend-kronos-staging-pr-1.herokuapp.com/)
- You can also view the [ah-kronos pipepline](https://dashboard.heroku.com/pipelines/654d28be-afe7-4b41-9f4a-08041637a794) to the active apps in the pipeline

#### What are the relevant pivotal tracker stories?
[#166880030](https://www.pivotaltracker.com/story/show/166880030)

#### Screenshots
**Heroku deployment of staging app.**
Screenshot 2019-07-01 at 19 31 38

**Heroku deployment of review app.**
Screenshot 2019-07-01 at 19 57 22

**Heroku deployment pipeline.**
Screenshot 2019-07-01 at 19 57 48

### CODE REVIEW

File: config/urls.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 from rest_framework.documentation import include_docs_urls
 from rest_framework_swagger.views import get_swagger_view
 
-core_schema_view = include_docs_urls(title='AH Haven Space Sprinters API')
-schema_view = get_swagger_view(title='AH Haven Space Sprinters API')
+core_schema_view = include_docs_urls(title='Authors Haven Kronos API')
+schema_view = get_swagger_view(title='Authors Haven Kronos API')
```
Comment (Manorlds-Eaglespark) [inline config/urls.py:9]: Do we have Swagger Documentation for the API?

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks @Rhytah for having our application deployed. However, Travis build is now failing. If you fix this it will be awesome.

**Comment by Rhytah** [comment]
> Thanks @Rhytah for having our application deployed. However, Travis build is now failing. If you fix this it will be awesome.

@refred Thanks for the feedback but unfortunately, the changes you are requesting are outside my current story scope and would be inappropriate to try and resolve them on a PR that is meant to do something else entirely.

**Review by Manorlds-Eaglespark — CHANGES_REQUESTED** [review]

**Comment by reifred** [comment]
> > Thanks @Rhytah for having our application deployed. However, Travis build is now failing. If you fix this it will be awesome.
> 
> @refred Thanks for the feedback but unfortunately, the changes you are requesting are outside my current story scope and would be inappropriate to try and resolve them on a PR that is meant to do something else entirely.

Thanks for letting me know about this but why I requested was because I am not sure if a PR can be merged with failing tests.

File: config/urls.py (hunk 1 above)
Comment (Rhytah) [reply config/urls.py:9]: @manorlds-Eaglespark Yes we do

**Review by Manorlds-Eaglespark — APPROVED** [review]
All tests are passing and the integration seems to be working well.

**Review by reifred — APPROVED** [review]
Thanks @Rhytah, for working on the feedback. The build is now running.

**Review by Emmanuel-Dominic — APPROVED** [review]

**Pull request merged**